Codeinwp/neve
# Update starter content theme mods & theme screenshot

### Summary
- Updates starter content theme mods;
- Change theme screenshot


### Will affect visual aspect of the product

NO

### Test instructions


- The screenshot should be changed;
- Create a clean wp instance or one where the `fresh_site` option is set to `0`;
- The starter content customizer settings should be the same as on demosites: https://demosites.io/web-agency-gb/;
- From what I understand from @mghenciu, no content changes are necessary - the demo used to have some different content from the starter content before;


Closes Codeinwp/neve-pro-addon#1646.
